persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/ConnectionDAOImpl.kt

```diff
@@ -19,6 +20,57 @@ private class ConnectionMapper {
         toId = state.to_id,
         shouldNotify = state.should_notify
     )
+
+    fun toModel(
```

Comment:
to what model ? to me name is a bit vague

Reply:
it's a private function and it's only accessible in the mapper, we had the toModel with the object I just added a new one. we have this pattern everywhere. didn't want to break the convention.